cmd/main.go

```diff
@@ -34,6 +35,7 @@ func main() {
 		cli.WithPlugins(
 			&pluginv2.Plugin{},
 			&pluginv3.Plugin{},
+			&goconfiggenv1.Plugin{},
```

Comment:
@estroz regarding`kubebuilder init --plugins go-config-gen`. After https://github.com/kubernetes-sigs/kubebuilder/pull/2060 get merged we will be able to use it as `kubebuilder init --plugins=config-gen/alphav1,go/v3`  instead. 

See that, we will probably need to create customize plugins to add things on top via its transformer func and use the bundle plugin which will be like `config-gen/alphav1,go/v3,go.transform/alphav1` which that can be `go-config-gen/alphav1` as suggested. I think is that the reason for you choose the name. just to be sure :-)